src/components/KeyboardAvoidingView/index.js

```diff
@@ -1,18 +0,0 @@
-/*
- * This is a KeyboardAvoidingView only enabled for ios && disabled for all other platforms
```

Comment:
When this file was removed, it led to a lot of regressions by having Android use `<KeyboardAvoidingView>`. We found in https://github.com/Expensify/App/issues/11087 that it's best to not use `<KeyboardAvoidingView>` on Android at all.